realm/realm-library/src/main/cpp/io_realm_SyncSession.cpp

```diff
@@ -49,3 +50,64 @@ JNIEXPORT jboolean JNICALL Java_io_realm_SyncSession_nativeRefreshAccessToken(JN
     CATCH_STD()
     return JNI_FALSE;
 }
+
+
+JNIEXPORT jlong JNICALL Java_io_realm_SyncSession_nativeAddProgressListener(JNIEnv* env, jclass,
+                                                                            jstring localRealmPath, jlong listenerId,
```

Comment:
uh.. jni functions are c++ code so we should follow the c++ rules i think? (At least i am trying to follow it for last a few months .... )